src/cstar-cli/cstar/cli/bpd/paymentinstrument.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2
+import pandas as pd
+import uuid
+
+
+class Paymentinstrument():
+
+  def __init__(self, args):
+    self.args = args
+    self.db_connection = psycopg2.connect(args.connection_string)
+  
+  def enroll(self):
+    
+    payment_instruments_df = pd.read_csv(self.args.file, sep=';')
+    if payment_instruments_df.shape[1] != 4:
+      raise ValueError(f"Invalid number of column in input file (expected 4, found {payment_instruments_df.shape[1]})")
+
+    insert_id = uuid.uuid4()
+
+    cursor = self.db_connection.cursor()
+
+    for index, row in payment_instruments_df.iterrows():
+
+      print(f"Processing row {index}")
+
+      insert_payment_instrument_q = cursor.mogrify(
```

Comment:
@alberto-re-pagopa Maybe we could use the method `execute_batch(...)` and remove the `for` cycle